stages/apps/lead/variables.tf

```
@@ -56,6 +56,14 @@ variable "opa_failure_policy" {
   default = "Fail"
 }
 
+variable "enable_sparky" {
+  default = true
```

I want us to start using a disabled by default approach to feature flags. It makes it easier to see what is running in an environment by just looking at the inputs in the terragrunt.hcl instead of having to check what is enabled by default.
```suggestion
  default = false
```

---

that makes sense to me